jest.setup.ts

```diff
@@ -1 +1,3 @@
 import '@testing-library/jest-dom/extend-expect'
+
+jest.setTimeout(30000);
```

Comment:
Why?

Reply:
some test that forgot to resolve async or to close the db can run forever

Reply:
Oh then this is cool